eth/backend.go

```diff
@@ -84,17 +82,33 @@ func (b *backend) SendTransaction(ctx context.Context, tx *types.Transaction) er
 	b.nonceManager.Unlock(sender)
 
 	data := tx.Data()
-	method, ok := b.methods[string(data[:4])]
-	if !ok {
-		method = "unknown"
+	methodSig := string(data[:4])
+	method, err := b.abiMap[methodSig].MethodById(data[:4])
+	if err != nil {
+		return err
+	}
+	var methodName string
+	if method == nil {
```

Comment:
If we're already checking the error returned by `MethodById()` on L87, I don't think we need this nil check for `method`.

Reply:
fixed in b47e031